frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts

```diff
@@ -35,6 +35,7 @@ import {Highlighting} from "core-components/wp-fast-table/builders/highlighting/
 import {HalResource} from "core-app/modules/hal/resources/hal-resource";
 import {WorkPackageCacheService} from "core-components/work-packages/work-package-cache.service";
 import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {WorkPackageChangeset} from "core-components/wp-edit/work-package-changeset";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/3e56c00ac126880e54f52ed25db2d574d6e8b35a/frontend/src/app/components/wp-buttons/wp-status-button/wp-status-button.component.ts#L39"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>